Configuration/Settings.yaml

```diff
@@ -22,8 +25,6 @@ Neos:
         'Neos.GoogleAnalytics': TRUE
 
   GoogleAnalytics:
-    enableTracking: true
```

Comment:
Can you provide upgrade instructions in the readme/docs which setting people should use instead of `enableTracking`?